packages/frontend/src/components/FlowStep/index.tsx

```diff
@@ -218,10 +218,14 @@ export default function FlowStep(
       iconUrl={app?.iconUrl}
       caption={
         displayOverrides?.caption ??
-        (app?.name ? `${step.position}. ${app.name}` : 'Choose an app')
+        (app?.name
+          ? `${step.position}. ${app.name}`
+          : isTrigger
+          ? 'This step starts your pipe'
+          : 'This step happens after your pipe starts')
```

Comment:
Thinking we can add even more clarity. Something like

```
let caption = ''
  if (app?.name) {
    caption = `${step.position}. ${app.name}`
  } else if (isTrigger) {
    caption = 'This step starts your pipe'
  } else if (step.position === 2) {
    caption = 'This step happens after your pipe starts'
  } else {
    caption = 'This step happens after the previous step'
  }
```

Then 
```
<FlowStepHeader
      iconUrl={app?.iconUrl}
      caption={displayOverrides?.caption ?? caption}
```

Reply:
ok boss thks i wanted something similar, got stacey approval too hehe